docs/basics/installation.rst

```diff
@@ -1,41 +1,86 @@
 .. _install:
 
-******************************
 Installation and Configuration
-******************************
+------------------------------
 
 Install DataLad
-===============
+^^^^^^^^^^^^^^^
 
-The content on this site is copied from the `DataLad website <https://www.datalad.org/get_datalad.html>`_
+The content on this site is largely based on
+`DataLad website <https://www.datalad.org/get_datalad.html>`_
 and the `DataLad documentation <http://docs.datalad.org/en/latest/gettingstarted.html>`_.
 
-When there isn't anything more convenient
------------------------------------------
+If you are not on a Debian-based system with Neurodebian enabled (see below),
+beyond DataLad itself, the installation requires Python, Pythons package manager ``pip``,
+:term:`Git` and :term:`Git-annex`. The instructions below detail how to install
+each of these components for different common operating systems. Please
+`file an issue <https://github.com/datalad-handbook/book/issues/new>`_
+if you encounter problems.
 
-When there isn't anything more convenient (i.e. availability via package managers; see below),
-DataLad is most easily installed via
-`pip <https://pip.pypa.io/en/stable/>`_.
-To automatically install DataLad and its software dependencies type
+
+Linux: (Neuro)Debian, Ubuntu, and similar systems
+"""""""""""""""""""""""""""""""""""""""""""""""""
+
+For Debian-based operating systems, the most convenient installation method
+is to enable the `NeuroDebian <http://neuro.debian.net/>`_ repository.
+If you are on a Debian-based system, but do not have the NeuroDebian repository
```

Comment:
Maybe add a note that this will do no harm to non-neuro people that need no neuro-software, despite the name.